test/e2e/vsphere_volume_placement.go

```diff
@@ -19,218 +19,412 @@ package e2e
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 	vsphere "k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere"
 	"k8s.io/kubernetes/test/e2e/framework"
+	"os"
+	"strconv"
 )
 
 var _ = framework.KubeDescribe("Volume Placement [Feature:Volume]", func() {
```

Comment:
Same comment from earlier reviews on use of `[Volume]` vs `[Feature:Volume]`? I currently see:
0 ocurrences of "[Feature:Volume]"
1 occurrence of "[Feature:Volumes]"
0 occurrences of "[Volumez]" and
80+ occurrences of  "[Volume[]".